server/datastore/mysql/migrations/tables/20240925112748_AddSetupExperienceResultsTable.go

```diff
@@ -54,7 +54,6 @@ ALTER TABLE host_script_results
 CREATE TABLE setup_experience_status_results (
 	id		INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
 	host_uuid	VARCHAR(255) COLLATE utf8mb4_unicode_ci NOT NULL,
-	type		ENUM('bootstrap-package', 'software-install', 'post-install-script') NOT NULL,
```

Comment:
The idea is that this will be implicit depending on the IDs below that are non-null? And we don't care about the bootstrap package anymore here because it happens before that setup experience dialog?

Reply:
exactly @mna !